api/src/commonMain/kotlin/com/buenaflor/kflogger/Klass.kt

```diff
@@ -0,0 +1,7 @@
+package com.buenaflor.kflogger
+
+import kotlin.reflect.KClass
+
+public expect class Klass<T : Any>
```

Comment:
Do you want to define some properties, functions, or extension functions here? It's unclear what benefits you get by making this class an expected class -- although it looks like the usages are opaque to callers outside KFlogger so maybe this is okay.

Reply:
Usages won't be really a concern outside of KFlogger, the reason why it's an expected class is because it typealiases directly on JVM to Class and on iOS it saves the reference to the KClass. It's basically a "bridge" class without functionality just so we can keep the Java typealias.